Algorithms/Sort/SelectionSort/README.md

```diff
@@ -0,0 +1,37 @@
+# Selection Sort
+Selection sort is a straightforward sorting algorithm that sorts an array by repeatedly finding the minimum element from the unsorted part of the array and putting it at the beginning. The process is iteratively applied to the remaining unsorted elements until the entire array is sorted.
+
+## Selection Sort
+```
+function selectionSort(array):
+    n = length(array)
```

Comment:
> [!Note]
> ## Reference from STANDARD.md
> - **Data Structure Operations:**
>  - Arrays/Lists:
>    - Access: `array[index]`
>    - Length: `array.length()`
>    - Insertion: `array.append(element)`
>    - Deletion: `array.remove(element)`

As that states you should use `array.length()` instead of `length(array)`